scipy/stats/tests/test_continuous_fit_censored.py

```diff
@@ -0,0 +1,463 @@
+# Tests for specific distributions of fitting censored data and
+# computing standard errors for the fitted parameters.
+
+import numpy as np
```

Comment:
Add generic test in `test_continuous_basic.py`.

Reply:
Tests for `CensoredData` and generic tests along the lines that you suggested are in progress.

Reply:
I added an additional check in the function `test_cont_fit` in `test_fit.py`, so now the MLE fit is computed twice, once with an array and then again with that array in a `CensoredData` instance.  The test checks that the two results are the same.  This is a bit trivial, because I also modified the `fit` method so that if a `CensoredData` instance is given that has no censored data, `fit` pulls the array out of the `CensoredData` object and does the fit with a regular array.

I haven't found a good set of generic tests that we can apply to an arbitrary distribution with nontrivial censored data that has a known correct result.

Reply:
If you generate random variates from a distribution, the fit parameters tend to be "close" to the parameters of the distribution. That shouldn't change much if you put small intervals around some (or all) of the variates, right?

Reply:
Good idea.  In the update that I just pushed, I added a generic test that converts the first 20 samples of the data used to test `fit` to interval-censored samples.  I experimented with the number of interval-censored values, the size of the intervals, and the relative tolerance for the comparison to the result of fitting the uncensored data.  With the values I ended up committing, there are still three distributions that don't pass the test: `chi`, `genhalflogistic` and `rdist`, so for now I just don't run the censored data on these distributions.  If I use different parameters--as defined in `_distr_params.py`--for these distributions (e.g. 15 for `chi`, 0.5 for `genhalflogistic` and 3.2 for `rdist`), the tests pass.